solanches/rest.py

```diff
@@ -218,6 +218,47 @@ def remove_produto(comercio_nome, produto_id):
     return jsonify(cardapio), 200
 
 
+@app.route("/comercio/<comercio_nome>/destaques/<produto_id>", methods=['DELETE'])
+def remove_produto_destaques(comercio_nome, produto_id):
+    try:
+        cardapio = controller.remove_produto_destaques(comercio_nome, produto_id)
+    except Exception as error:
+        _assert(False, 400, str(error))
+
+    return jsonify(cardapio), 200
+
+
+@app.route("/comercio/<comercio_nome>/categoria", methods=['POST'])
+def adiciona_categoria(comercio_nome):
+    req = request.get_json()
+    _assert (req, 400, "Erro: json inválido!")
+    categoria = req.get("categoria")
+    _assert (categoria, 400, "Erro: categoria não informada!")
+
+    try:
+        controller.adiciona_categoria(comercio_nome, categoria)
+        msg = {"message": "Categoria adicionada!"}
+    except Exception as error:
+        _assert(False, 400, str(error))
+
+    return jsonify(msg), 201
```

Comment:
Acho que é bom retornar o cardápio atualizado
```suggestion
        cardapio_atualizado = controller.adiciona_categoria(comercio_nome, categoria)
    except Exception as error:
        _assert(False, 400, str(error))

    return jsonify(cardapio_atualizado), 201
```